src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,29 @@
 package mate.academy.hibernate.relations.util;
 
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            Configuration configuration = new Configuration().configure();
+            configuration.setProperty("hibernate.connection.username",
+                    System.getenv("DB_USER"));
+            configuration.setProperty("hibernate.connection.password",
+                    System.getenv("DB_PASSWORD"));
+            return configuration.buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcessingException("Can`t create session factory", e);
```

Comment:
Ensure that the environment variables `DB_USER` and `DB_PASSWORD` are set correctly in the environment where this application is running. If these variables are not set, the `SessionFactory` initialization will fail, leading to a `DataProcessingException`. Consider adding a check or a default value for these properties to avoid runtime errors.